src/metabase/util/grouper.clj

```diff
@@ -0,0 +1,34 @@
+(ns metabase.util.grouper
+  "Our wrapper for grouper -- the batch processing utility.
+
+  Note:
+  - These utilities should only be used for scenarios where data consistency is not a requirement,
```

Comment:
We won't even get eventual consistency, so I think it's worth rewording this that execution is "best effort", and may not even occur.